pkgs/development/python-modules/buildbot/worker.nix

```diff
@@ -3,11 +3,11 @@
 
 buildPythonPackage (rec {
   pname = "buildbot-worker";
-  version = "2.8.0";
+  version = "2.8.1";
```

Comment:
Here as well?

Reply:
I guess so. I don't think it is as important here because there is not as tight a coupling between the master and worker as there is between the master and its plugins.